src/main/webapp/user-page.html

```diff
@@ -20,6 +20,8 @@
     <meta charset="UTF-8">
     <title>User Page</title>
     <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <script src="/js/user-page-loader.js"></script>
     <link href="https://fonts.googleapis.com/css?family=Teko&display=swap" rel="stylesheet">
   </head>
   </head>
```

Comment:
I think you might have a duplicate `</head>` tag here. You should remove one of them.

Reply:
ok